### ISSUE TITLE
More Compact View

### ISSUE THREAD

**Issue by hankcarr** [liftoff-app/liftoff#226]
### Is your feature request related to a problem? Please describe.

I find the compact list view to busy. 

### Describe the solution you'd like

I would like the option to hide the first couple of lines of stuff, who wrote it, where it came from, etc in the list view.  This would make each post a title and the bottom stuff. The other stuff is available once you click into the post anyway.

**Comment by zachatrocity** [comment]
I personally want this as well. I'll be adding an ultra compact

**Comment by jordan1776** [comment]
Yeah agree, I feel like there should be some kind of list view (which doesn't have the card motif). Compact view fits about 5 posts as compared to other aps whose ultra compact/list views fit 7-8 posts. @battaglia01 looks to have some good ideas in #222. I think hiding the interaction bar could also save some space and have those hidden behind a long press or implement #69 might be one way to go about it but we'll see.
